ws-nextjs-app/pages/_document.page.tsx

```diff
@@ -24,11 +71,14 @@ type DocProps = {
 export default class AppDocument extends Document<DocProps> {
   static async getInitialProps(ctx: DocumentContext) {
     const initialProps = await Document.getInitialProps(ctx);
-    const isApp = isAppPath(ctx.asPath || '');
+    const url = ctx.asPath || '';
+    const isApp = isAppPath(url);
 
     // Read env variables from the server and expose them to the client
     const clientSideEnvVariables = getProcessEnvAppVariables();
 
+    handleServerLogging(ctx);
```

Comment:
Does this code get bundled in the client-side app?

Reply:
No I don't think it should. Its not returned in the `getInitialProps` object and this function should only ever run on the server.